airflow/www/views.py

```diff
@@ -121,6 +121,7 @@
 from airflow.utils.state import State
 from airflow.utils.strings import to_boolean
 from airflow.utils.timezone import td_format
+from airflow.utils.timezone import utcnow
```

Comment:
```suggestion
from airflow.utils.timezone import td_format, utcnow
```

To fix static checks